tests/snapshots/test_snapshots/test_ensemble_smoother_update_step/test_ensemble_smoother_update_step.csv

```diff
@@ -1,3 +1,3 @@
-1.967146767000000018e+00,1.887802729999999984e+00,1.521310943000000027e+00,5.066928569999999965e-01,1.386576888000000007e+00,1.430903387000000082e+00,8.768934029999999602e-01,1.026742226999999952e+00,1.841859796000000049e+00,2.096290783999999796e+00
-5.100308150000000262e-01,-7.912242240000000315e-01,1.429665179999999869e-01,3.900854710000000170e-01,-7.375762200000000046e-01,-8.464409999999999989e-02,-1.366024670000000052e-01,-3.888128559999999845e-01,-4.576220990000000044e-01,7.570495100000000643e-02
--1.943097159999999934e-01,5.865930440000000079e-01,-9.175710600000000494e-02,3.380177240000000194e-01,1.868880370000000068e-01,-8.676134999999999750e-03,-4.114721799999999929e-02,2.734158270000000002e-01,5.822034710000000279e-01,-2.447357490000000024e-01
+1.571474030999999938e+00,2.053739563000000157e+00,1.527342551999999909e+00,1.361049552999999968e+00,1.466300458999999945e+00,1.053619995999999892e+00,1.387315756000000011e+00,8.378489999999999549e-01,1.283992301999999919e+00,1.999536568999999986e+00
```

Comment:
Have not read through the changes carefully yet.
What causes the need to update snapshots?

Reply:
Just the fact that using `rng` and sampling from normal/MVnormal was not identical to generating `noise`. Functionally it should be the same

Reply:
Am I understanding correctly that using `multivariate_normal` with method set to "cholesky" is different from doing it ourselves?

Reply:
Yes, but only different in the sense that it's like using a different seed.